### PR TITLE
Gdv 227 shoreline timeseries

### PR DESCRIPTION
**Pull request by evetion**

### CODE REVIEW

File: dgds_backend/dgds_shoreline_service.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    transect["properties"]["changeRateUnc"] = feature["properties"]["change_rate_unc"]
+    transect["startTime"] = dates[0].strftime("%Y-%m-%dT%H:%M:%SZ")
+    transect["endTime"] = dates[-1].strftime("%Y-%m-%dT%H:%M:%SZ")
+    transect["events"] = events_list
+
+    return template.render(dataset=dataset, transect=transect)
+
+
+def dd_shoreline(url, transect_id, dataset_name, dataset_id):
+    box, section = transect_id.split("_")
```
Comment (ckrogers) [inline dgds_backend/dgds_shoreline_service.py]: The transect_id gas a BOX_box#_section#_transect#, but files are in format BOX_box#_section#. The specific transect number should be added, but is not needed for the url.

File: dgds_backend/app.py [hunk 2]
```diff
@@ -149,18 +168,14 @@ def locations():
...
 
     # Query PiService
     pi = PiServiceDDL(observation_type_id, pi_service_url, request.url_root)
-    content = {}
-    try:
-        content = pi.get_locations(input)
-    except Exception as e:
-        content = {'error': 'The PiService-DDL failed to serve the response. Please try again later'}
-        logging.error('The PiService-DDL failed to serve the response. Please try again later')
+    content = pi.get_locations(input)
```
Comment (ckrogers) [inline dgds_backend/app.py]: 😎

File: dgds_backend/config_data/datasets_access.json [hunk 3]
```diff
@@ -111,9 +111,9 @@
   },
   "sm": {
     "dataService":  {
-      "url": "",
-      "name": "",
-      "protocol": "",
+      "url": "https://storage.googleapis.com/shoreline-monitor/features/{box}/{section}/BOX_{box}_{section}.json",
```
Comment (ckrogers) [inline dgds_backend/config_data/datasets_access.json]: 👍

**Comment by evetion** [comment]
Still needs a primary key field set in one of the config files after discussion with frontend.

**Pull request merged**